### PR TITLE
[*] new from/to tiptap converters

### PR DESCRIPTION
**Pull request by lxgreen**

### CODE REVIEW

File: packages/plugin-hashtag/web/src/tiptap/tiptap.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
+import type { Group, RicosExtension } from 'ricos-tiptap-types';
 import findHashTag from './findHashTag';
 
-export const tiptapExtensions = [
+export const tiptapExtensions: RicosExtension[] = [
   {
     type: 'mark' as const,
-    groups: [],
+    groups: [] as Group[],
     name: 'hashtag',
```
Comment (maimonav) [inline packages/plugin-hashtag/web/src/tiptap/tiptap.ts:8]: No hashtag value in the schema, do we want to make it Constant just to be consistent?

File: packages/ricos-content/web/src/consts.ts [hunk 2]
```diff
@@ -123,22 +123,22 @@ export const RICOS_EXTERNAL_LINK_TYPE = 'ricos-external-link'; // TODO: is this
...
+export const TIPTAP_AUDIO_TYPE = Node_Type.AUDIO;
+export const TIPTAP_LINK_PREVIEW_TYPE = Node_Type.LINK_PREVIEW;
+export const TIPTAP_EMBED_TYPE = Node_Type.EMBED;
+export const TIPTAP_POLL_TYPE = Node_Type.POLL;
+export const TIPTAP_APP_EMBED_TYPE = Node_Type.APP_EMBED;
+export const TIPTAP_COLLAPSIBLE_LIST_TYPE = Node_Type.COLLAPSIBLE_LIST;
+export const TIPTAP_COLLAPSIBLE_ITEM_TYPE = Node_Type.COLLAPSIBLE_ITEM;
+export const TIPTAP_COLLAPSIBLE_ITEM_TITLE_TYPE = Node_Type.COLLAPSIBLE_ITEM_TITLE;
+export const TIPTAP_COLLAPSIBLE_ITEM_BODY_TYPE = Node_Type.COLLAPSIBLE_ITEM_BODY;
+export const TIPTAP_HTML_TYPE = Node_Type.HTML;
```
Comment (maimonav) [inline packages/ricos-content/web/src/consts.ts:141]: We need to remove this @lxgreen , but in other PR for sure

File: packages/tiptap-editor/web/src/components/RichContentAdapter/RichContentAdapter.tsx [hunk 3]
```diff
@@ -245,16 +246,16 @@ export class RichContentAdapter
       isBlockTypeSelected: type => {
         const blockTypeActiveCommandMap = {
           [UNSTYLED]: () => this.tiptapEditor.isActive('unstyled'),
```
Comment (maimonav) [inline packages/tiptap-editor/web/src/components/RichContentAdapter/RichContentAdapter.tsx:248]: What is `unstyled` in tiptap? shouldn't it be "PARAGRAPH"? @lxgreen

File: packages/plugin-hashtag/web/src/tiptap/tiptap.ts (hunk 1 above)
Comment (lxgreen) [reply packages/plugin-hashtag/web/src/tiptap/tiptap.ts:8]: yep, good catch

File: packages/tiptap-editor/web/src/components/RichContentAdapter/RichContentAdapter.tsx (hunk 3 above)
Comment (maimonav) [reply packages/tiptap-editor/web/src/components/RichContentAdapter/RichContentAdapter.tsx:248]: Nevermind, not being used.
Soon we will have a lot of code to remove :)

**Pull request merged**